starport/services/network/queries.go

```diff
@@ -35,7 +41,13 @@ func (n Network) ChainLaunches(ctx context.Context) ([]networktypes.ChainLaunch,
 
 	// Parse fetched chains
 	for _, chain := range res.Chain {
-		chainLaunches = append(chainLaunches, networktypes.ToChainLaunch(chain))
+		chainLaunch := networktypes.ToChainLaunch(chain)
+		reward, err := n.ChainReward(ctx, chain.LaunchID)
```

Comment:
I really like the event messages sent one by one through this func in format of "Fetching rewards for chain 30...".

But that's pretty time consuming. Can we use errgroup to concurrently do this? While fetching reward info, we can just send a single, general event to inform the user.

Let's make sure to set the proper ctx for it so when user do Control^C, all these can be canceled.